ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -123,8 +126,10 @@
    */
   private void initialize() throws Exception {
     for (DcZkInfo dcZkInfo : dcToDcZkInfo.values()) {
+      logger.info("Initializing cluster information from {}", dcZkInfo.zkConnectStr);
```

Comment:
won't it be useful to print messages before and after here (just like the connect)